webapp/src/scriptmanager.tsx

```diff
@@ -318,6 +318,17 @@ export class ScriptManagerDialog extends data.Component<ScriptManagerDialogProps
         this.setState({ sortedAsc: !sortedAsc });
     }
 
+    renderDownloadDialog(fileName: string) {
+        return <>
+            <div>
+                {lf("Your projects were zipped and downloaded successfully!")}
+                <br />
+                <br />
+                {lf("Look for the file {0} on your computer. It might have a number before the .zip if you've downloaded before.", fileName)}
```

Comment:
At least when I've done this, I get a popup where I can choose the location and name for the file (both in the browser and when running dev Minecraft build on Windows pointing to your uploaded target). I'm wondering if we really need this additional string, or if we can just say it was downloaded successfully and leave it at that?

Reply:
Yeah, that's what happens for me, too. I added the extra string because it just felt a bit bare to me with just one line, but I can remove the extra stuff.

Reply:
I prefer  concise language here. This needs to be translated to lot of languages and number system might be different. Just say "Your projects successfully downloaded as a zip"